### PR TITLE
[KOGITO-9315] Add Kogito Kubernetes extension to swf images

### PR DESCRIPTION
**Pull request by ricardozanini**
Many thanks for submitting your Pull Request :heart:! 

See: https://issues.redhat.com/browse/KOGITO-9315

Depends on:

- https://github.com/kiegroup/kogito-runtimes/pull/3056

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- <b>(Re)run Jenkins tests</b>  
  Please add comment: <b>Jenkins [test|retest] this</b>

- <b>Prod tests</b>  
  Please add comment: <b>Jenkins (re)run [prod|Prod|PROD]</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by fjtirado — APPROVED** [review]

File: modules/kogito-swf/builder/build-config/module.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ envs:
   - name: "SCRIPT_DEBUG"
     value: "false"
   - name: QUARKUS_EXTENSIONS
-    value: quarkus-kubernetes,kogito-quarkus-serverless-workflow,kogito-addons-quarkus-knative-eventing,smallrye-health
+    value: quarkus-kubernetes,kogito-quarkus-serverless-workflow,kogito-addons-quarkus-knative-eventing,smallrye-health,org.kie.kogito:kogito-addons-quarkus-fabric8-kubernetes-service-catalog:${KOGITO_VERSION},org.kie.kogito:kogito-addons-quarkus-kubernetes:${KOGITO_VERSION}
```
Comment (radtriste) [inline modules/kogito-swf/builder/build-config/module.yaml:12]: Please create a follow-up issue to update this

**Review by radtriste — APPROVED** [review]
minor comment

File: modules/kogito-swf/builder/build-config/module.yaml (hunk 1 above)
Comment (radtriste) [reply modules/kogito-swf/builder/build-config/module.yaml:12]: and nice catch :)

**Review by spolti — APPROVED** [review]

File: modules/kogito-swf/builder/build-config/module.yaml (hunk 1 above)
Comment (radtriste) [reply modules/kogito-swf/builder/build-config/module.yaml:12]: https://issues.redhat.com/browse/KOGITO-9270

**Comment by kie-ci1** [comment]
**PR job** `#861` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/861/display/redirect

**Test results:**
- PASSED: 269
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/861/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Perform_a_second_incremental_s2i_build_for_native_scenario,_this_time,_with_native_enabled/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Perform a second incremental s2i build for native scenario, this time, with native enabled</a></summary>
Phrase 'Expanding artifacts from incremental build...' was not found in the output of S2I
</details>

**Comment by ricardozanini** [comment]
jenkins retest this

**Comment by kie-ci1** [comment]
**PR job** `#869` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Build failures on those images:
- kogito-s2i-builder


Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/869/display/redirect

**Test results:**
- PASSED: 231
- FAILED: 0

Those are the test failures: none

See console log:
<details>
<summary><b>kogito-s2i-builder</b></summary>
        <repository><br/>          <id>jboss-public-repository-group</id><br/>          <name>JBoss Public Repository Group</name><br/>          <url>https://repository.jboss.org/nexus/content/groups/public/</url><br/>          <layout>default</layout><br/>          <releases><br/>            <enabled>true</enabled><br/>            <updatePolicy>never</updatePolicy><br/>          </releases><br/>          <snapshots><br/>            <enabled>true</enabled><br/>            <updatePolicy>always</updatePolicy><br/>          </snapshots><br/>        </repository><br/>        <br/>      </repositories><br/><br/>      <pluginRepositories><br/>        <pluginRepository><br/>          <id>jboss-public-repository-group</id><br/>          <name>JBoss Public Repository Group</name><br/>          <url>https://repository.jboss.org/nexus/content/groups/public/</url><br/>          <layout>default</layout><br/>          <releases><br/>            <enabled>true</enabled><br/>            <updatePolicy>never</updatePolicy><br/>          </releases><br/>          <snapshots><br/>            <enabled>true</enabled><br/>            <updatePolicy>always</updatePolicy><br/>          </snapshots><br/>        </pluginRepository><br/>        <br/>      </pluginRepositories><br/>    </profile><br/>  </profiles><br/>  <activeProfiles><br/>    <br/><activeProfile>internal-npm-registry</activeProfile><br/>    <activeProfile>kogito-images</activeProfile><br/>  </activeProfiles><br/></settings><br/>Copy current maven repo to maven context local repo /tmp/temp_maven/jobs-service-postgresql<br/>#2 DONE 6.4s<br/><br/>#3 [internal] load metadata for registry.access.redhat.com/ubi8/ubi-minimal:latest<br/><br/>#4 [internal] load build context<br/><br/>#3 [internal] load metadata for registry.access.redhat.com/ubi8/ubi-minimal:latest<br/>Updating settings file /tmp/trusty-service-redis/settings.xml<br/>INFO Configure Maven<br/>enabling npm repository: https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/<br/><settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"<br/>          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"<br/>          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0<br/>                      https://maven.apache.org/xsd/settings-1.0.0.xsd"><br/><br/>  <mirrors><br/>        <mirror>      <id>mirror.default</id>      <url>https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/groups/kogito-runtimes-pr-artifacts-group/</url>      <mirrorOf>external:*</mirrorOf>    </mirror><br/>  </mirrors><br/><br/>  <proxies><br/>    <br/>  </proxies><br/><br/>  <profiles><br/>    <br/><profile><id>internal-npm-registry</id><properties><npmRegistryURL>https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/</npmRegistryURL><yarnDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/yarn/</yarnDownloadRoot><nodeDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/node/</nodeDownloadRoot><npmDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/npm/</npmDownloadRoot></properties></profile><br/><br/>    <profile><br/>      <id>kogito-images</id><br/>      <repositories><br/>        <repository><br/>          <id>jboss-public-repository-group</id><br/>          <name>JBoss Public Repository Group</name><br/>          <url>https://repository.jboss.org/nexus/content/groups/public/</url><br/>          <layout>default</layout><br/>          <releases><br/>            <enabled>true</enabled><br/>            <updatePolicy>never</updatePolicy><br/>          </releases><br/>          <snapshots><br/>            <enabled>true</enabled><br/>            <updatePolicy>always</updatePolicy><br/>          </snapshots><br/>        </repository><br/>        <br/>      </repositories><br/><br/>      <pluginRepositories><br/>        <pluginRepository><br/>          <id>jboss-public-repository-group</id><br/>          <name>JBoss Public Repository Group</name><br/>          <url>https://repository.jboss.org/nexus/content/groups/public/</url><br/>          <layout>default</layout><br/>          <releases><br/>            <enabled>true</enabled><br/>            <updatePolicy>never</updatePolicy><br/>          </releases><br/>          <snapshots><br/>            <enabled>true</enabled><br/>            <updatePolicy>always</updatePolicy><br/>          </snapshots><br/>        </pluginRepository><br/>        <br/>      </pluginRepositories><br/>    </profile><br/>  </profiles><br/>  <activeProfiles><br/>    <br/><activeProfile>internal-npm-registry</activeProfile><br/>    <activeProfile>kogito-images</activeProfile><br/>  </activeProfiles><br/></settings><br/>Copy current maven repo to maven context local repo /tmp/temp_maven/trusty-service-redis<br/>Updating settings file /tmp/trusty-service-postgresql/settings.xml<br/>INFO Configure Maven<br/>enabling npm repository: https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/<br/><settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"<br/>          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"<br/>          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0<br/>                      https://maven.apache.org/xsd/settings-1.0.0.xsd"><br/><br/>  <mirrors><br/>        <mirror>      <id>mirror.default</id>      <url>https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/groups/kogito-runtimes-pr-artifacts-group/</url>      <mirrorOf>external:*</mirrorOf>    </mirror><br/>  </mirrors><br/><br/>  <proxies><br/>    <br/>  </proxies><br/><br/>  <profiles><br/>    <br/><profile><id>internal-npm-registry</id><properties><npmRegistryURL>https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/</npmRegistryURL><yarnDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/yarn/</yarnDownloadRoot><nodeDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/node/</nodeDownloadRoot><npmDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/npm/</npmDownloadRoot></properties></profile><br/><br/>    <profile><br/>      <id>kogito-images</id><br/>      <repositories><br/>        <repository><br/>          <id>jboss-public-repository-group</id><br/>          <name>JBoss Public Repository Group</name><br/>          <url>https://repository.jboss.org/nexus/content/groups/public/</url><br/>          <layout>default</layout><br/>          <releases><br/>            <enabled>true</enabled><br/>            <updatePolicy>never</updatePolicy><br/>          </releases><br/>          <snapshots><br/>            <enabled>true</enabled><br/>            <updatePolicy>always</updatePolicy><br/>          </snapshots><br/>        </repository><br/>        <br/>      </repositories><br/><br/>      <pluginRepositories><br/>        <pluginRepository><br/>          <id>jboss-public-repository-group</id><br/>          <name>JBoss Public Repository Group</name><br/>          <url>https://repository.jboss.org/nexus/content/groups/public/</url><br/>          <layout>default</layout><br/>          <releases><br/>            <enabled>true</enabled><br/>            <updatePolicy>never</updatePolicy><br/>          </releases><br/>          <snapshots><br/>            <enabled>true</enabled><br/>            <updatePolicy>always</updatePolicy><br/>          </snapshots><br/>        </pluginRepository><br/>        <br/>      </pluginRepositories><br/>    </profile><br/>  </profiles><br/>  <activeProfiles><br/>    <br/><activeProfile>internal-npm-registry</activeProfile><br/>    <activeProfile>kogito-images</activeProfile><br/>  </activeProfiles><br/></settings><br/>Copy current maven repo to maven context local repo /tmp/temp_maven/trusty-service-postgresql<br/>Updating settings file /tmp/trusty-service-infinispan/settings.xml<br/>INFO Configure Maven<br/>enabling npm repository: https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/<br/><settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"<br/>          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"<br/>          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0<br/>                      https://maven.apache.org/xsd/settings-1.0.0.xsd"><br/><br/>  <mirrors><br/>        <mirror>      <id>mirror.default</id>      <url>https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/groups/kogito-runtimes-pr-artifacts-group/</url>      <mirrorOf>external:*</mirrorOf>    </mirror><br/>  </mirrors><br/><br/>  <proxies><br/>    <br/>  </proxies><br/><br/>  <profiles><br/>    <br/><profile><id>internal-npm-registry</id><properties><npmRegistryURL>https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/</npmRegistryURL><yarnDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/yarn/</yarnDownloadRoot><nodeDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/node/</nodeDownloadRoot><npmDownloadRoot>http://download.devel.redhat.com/rcm-guest/staging/rhba/dist/npm/</npmDownloadRoot></properties></profile><br/><br/>    <profile><br/>      <id>kogito-images</id><br/>      <repositories><br/>        <repository><br/>          <id>jboss-public-repository-group</id><br/>          <name>JBoss Public Repository Group</name><br/>          <url>https://repository.jboss.org/nexus/content/groups/public/</url><br/>          <layout>default</layout><br/>          <releases><br/>            <enabled>true</enabled><br/>            <updatePolicy>never</updatePolicy><br/>          </releases><br/>          <snapshots><br/>            <enabled>true</enabled><br/>            <updatePolicy>always</updatePolicy><br/>          </snapshots><br/>        </repository><br/>        <br/>      </repositories><br/><br/>      <pluginRepositories><br/>        <pluginRepository><br/>          <id>jboss-public-repository-group</id><br/>          <name>JBoss Public Repository Group</name><br/>          <url>https://repository.jboss.org/nexus/content/groups/public/</url><br/>          <layout>default</layout><br/>          <releases><br/>            <enabled>true</enabled><br/>            <updatePolicy>never</updatePolicy><br/>          </releases><br/>          <snapshots><br/>            <enabled>true</enabled><br/>            <updatePolicy>always</updatePolicy><br/>          </snapshots><br/>        </pluginRepository><br/>        <br/>      </pluginRepositories><br/>    </profile><br/>  </profiles><br/>  <activeProfiles><br/>    <br/><activeProfile>internal-npm-registry</activeProfile><br/>    <activeProfile>kogito-images</activeProfile><br/>  </activeProfiles><br/></settings><br/>Copy current maven repo to maven context local repo /tmp/temp_maven/trusty-service-infinispan<br/>[37m2023-06-15 10:01:41,061 crypto.py:34         DEBUG Checking './cache/1d1e2db0-abef-4e16-b02b-28566b0e8073' md5 hash...[0m<br/>[37m2023-06-15 10:01:41,061 crypto.py:20         DEBUG Computing md5 checksum for './cache/1d1e2db0-abef-4e16-b02b-28566b0e8073' file[0m<br/>[37m2023-06-15 10:01:41,519 crypto.py:46         DEBUG Hash is correct.[0m<br/>[37m2023-06-15 10:01:41,519 crypto.py:20         DEBUG Computing sha512 checksum for './cache/1d1e2db0-abef-4e16-b02b-28566b0e8073' file[0m<br/><br/>#3 [internal] load metadata for registry.access.redhat.com/ubi8/ubi-minimal:latest<br/>#4 transferring context:<br/>#4 transferring context: 108.04kB 0.2s done<br/>#4 DONE 2.3s<br/>[37m2023-06-15 10:01:41,955 crypto.py:20         DEBUG Computing sha256 checksum for './cache/1d1e2db0-abef-4e16-b02b-28566b0e8073' file[0m<br/>#3 DONE 1.1s<br/><br/>#4 [ 1/29] FROM registry.access.redhat.com/ubi8/ubi-minimal:latest@sha256:621f5245fb3e8597a626163cdf1229e1f8311e07ab71bb1e9332014b51c59f9c<br/>#4 resolve registry.access.redhat.com/ubi8/ubi-minimal:latest@sha256:621f5245fb3e8597a626163cdf1229e1f8311e07ab71bb1e9332014b51c59f9c 0.7s done<br/>#4 DONE 0.9s<br/><br/>#5 [internal] load build context<br/>#3 DONE 2.5s<br/><br/>#4 [ 1/29] FROM registry.access.redhat.com/ubi8/ubi-minimal:latest@sha256:621f5245fb3e8597a626163cdf1229e1f8311e07ab71bb1e9332014b51c59f9c<br/>#4 resolve registry.access.redhat.com/ubi8/ubi-minimal:latest@sha256:621f5245fb3e8597a626163cdf1229e1f8311e07ab71bb1e9332014b51c59f9c 0.7s done<br/>#4 DONE 0.9s<br/><br/>#5 [internal] load build context<br/>[37m2023-06-15 10:01:42,253 crypto.py:20         DEBUG Computing sha1 checksum for './cache/1d1e2db0-abef-4e16-b02b-28566b0e8073' file[0m<br/>#3 DONE 2.2s<br/><br/>#4 [ 1/29] FROM registry.access.redhat.com/ubi8/ubi-minimal:latest@sha256:621f5245fb3e8597a626163cdf1229e1f8311e07ab71bb1e9332014b51c59f9c<br/>#4 resolve registry.access.redhat.com/ubi8/ubi-minimal:latest@sha256:621f5245fb3e8597a626163cdf1229e1f8311e07ab71bb1e9332014b51c59f9c 0.7s done<br/>#4 DONE 0.9s<br/><br/>#5 [internal] load build context<br/>[37m2023-06-15 10:01:42,496 crypto.py:20         DEBUG Computing md5 checksum for './cache/1d1e2db0-abef-4e16-b02b-28566b0e8073' file[0m<br/>#5 transferring context:<br/>#5 transferring context: 83.53kB 0.2s done<br/>#5 transferring context: 110B<br/>[32m2023-06-15 10:01:43,137 resource.py:243        INFO  Using cached artifact 'graalvm-ce-java11-linux-amd64-22.3.2.tar.gz'.[0m<br/>[32m2023-06-15 10:01:43,137 resource.py:226        INFO  Copying resource 'graalvm-ce-java11-linux-aarch64-22.3.2.tar.gz'...[0m<br/>[37m2023-06-15 10:01:43,138 resource.py:301        DEBUG Using md5 checksum to fetch artifacts from cacher[0m<br/>[37m2023-06-15 10:01:43,139 resource.py:311        DEBUG Using cache url 'http://ba-cekit-cacher.usersys.redhat.com/resource/5cb31954ef5538057d83502c19492ad0'[0m<br/>[37m2023-06-15 10:01:43,139 tools.py:39         DEBUG Downloading from 'http://ba-cekit-cacher.usersys.redhat.com/resource/5cb31954ef5538057d83502c19492ad0' as ./cache/feff0c84-c585-4eb3-a0f4-31e515e8c3ea[0m<br/>[37m2023-06-15 10:01:43,150 resource.py:465        DEBUG Cannot hit artifact: 'graalvm-ce-java11-linux-aarch64-22.3.2.tar.gz' via cache, trying directly.[0m<br/>[37m2023-06-15 10:01:43,150 tools.py:39         DEBUG Downloading from 'https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-22.3.2/graalvm-ce-java11-linux-aarch64-22.3.2.tar.gz' as ./cache/feff0c84-c585-4eb3-a0f4-31e515e8c3ea[0m<br/>Downloading graalvm-ce-java11-linux-aarch64-22.3.2.tar.gz<br/>#5 transferring context: 8.74MB 0.4s done<br/>#5 transferring context: 110B<br/>#5 transferring context: 8.74MB 0.3s done<br/>#5 DONE 3.9s<br/>#5 DONE 5.1s<br/>[37m2023-06-15 10:01:47,272 crypto.py:34         DEBUG Checking './cache/feff0c84-c585-4eb3-a0f4-31e515e8c3ea' md5 hash...[0m<br/>[37m2023-06-15 10:01:47,272 crypto.py:20         DEBUG Computing md5 checksum for './cache/feff0c84-c585-4eb3-a0f4-31e515e8c3ea' file[0m<br/>[37m2023-06-15 10:01:47,726 crypto.py:46         DEBUG Hash is correct.[0m<br/>[37m2023-06-15 10:01:47,727 crypto.py:20         DEBUG Computing sha512 checksum for './cache/feff0c84-c585-4eb3-a0f4-31e515e8c3ea' file[0m<br/>[37m2023-06-15 10:01:48,151 crypto.py:20         DEBUG Computing sha256 checksum for './cache/feff0c84-c585-4eb3-a0f4-31e515e8c3ea' file[0m<br/>[37m2023-06-15 10:01:48,392 crypto.py:20         DEBUG Computing sha1 checksum for './cache/feff0c84-c585-4eb3-a0f4-31e515e8c3ea' file[0m<br/>[37m2023-06-15 10:01:48,723 crypto.py:20         DEBUG Computing md5 checksum for './cache/feff0c84-c585-4eb3-a0f4-31e515e8c3ea' file[0m<br/>#5 DONE 6.4s<br/>[32m2023-06-15 10:01:49,320 resource.py:243        INFO  Using cached artifact 'graalvm-ce-java11-linux-aarch64-22.3.2.tar.gz'.[0m<br/>[32m2023-06-15 10:01:49,321 resource.py:226        INFO  Copying resource 'native-image-installable-svm-java11-linux-amd64-22.3.2.jar'...[0m<br/>[37m2023-06-15 10:01:49,323 resource.py:301        DEBUG Using md5 checksum to fetch artifacts from cacher[0m<br/>[37m2023-06-15 10:01:49,500 resource.py:311        DEBUG Using cache url 'http://ba-cekit-cacher.usersys.redhat.com/resource/dae44733baf47ffaadf7ef98959b5653'[0m<br/>[37m2023-06-15 10:01:49,531 tools.py:39         DEBUG Downloading from 'http://ba-cekit-cacher.usersys.redhat.com/resource/dae44733baf47ffaadf7ef98959b5653' as ./cache/1e0fd3af-e158-45a4-ae1a-63f62355843b[0m<br/>[37m2023-06-15 10:01:49,542 resource.py:465        DEBUG Cannot hit artifact: 'native-image-installable-svm-java11-linux-amd64-22.3.2.jar' via cache, trying directly.[0m<br/>[37m2023-06-15 10:01:49,542 tools.py:39         DEBUG Downloading from 'https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-22.3.2/native-image-installable-svm-java11-linux-amd64-22.3.2.jar' as ./cache/1e0fd3af-e158-45a4-ae1a-63f62355843b[0m<br/>Downloading native-image-installable-svm-java11-linux-amd64-22.3.2.jar<br/><br/>#6 [ 4/29] RUN [ "sh", "-x", "/tmp/scripts/org.kie.kogito.image.dependencies/clean.sh" ]<br/>#6 CACHED<br/><br/>#7 [ 5/29] COPY modules/org.kie.kogito.system.user /tmp/scripts/org.kie.kogito.system.user<br/>#7 CACHED<br/><br/>#8 [ 6/29] RUN [ "sh", "-x", "/tmp/scripts/org.kie.kogito.system.user/add-user" ]<br/>#8 CACHED<br/><br/>#9 [ 7/29] COPY modules/org.kie.kogito.logging /tmp/scripts/org.kie.kogito.logging<br/>#9 CACHED<br/><br/>#10 [ 2/29] COPY modules/org.kie.kogito.image.dependencies /tmp/scripts/org.kie.kogito.image.dependencies<br/>#10 CACHED<br/><br/>#11 [ 3/29] RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y file shadow-utils tar gzip unzip             && microdnf clean all             && rpm -q file shadow-utils tar gzip unzip<br/>#11 CACHED<br/><br/>#12 [ 8/29] RUN [ "sh", "-x", "/tmp/scripts/org.kie.kogito.logging/configure" ]<br/>#12 CACHED<br/><br/>#13 [ 9/29] COPY modules/org.kie.kogito.launch.scripts /tmp/scripts/org.kie.kogito.launch.scripts<br/>[37m2023-06-15 10:01:50,335 crypto.py:34         DEBUG Checking './cache/1e0fd3af-e158-45a4-ae1a-63f62355843b' md5 hash...[0m<br/>[37m2023-06-15 10:01:50,336 crypto.py:20         DEBUG Computing md5 checksum for './cache/1e0fd3af-e158-45a4-ae1a-63f62355843b' file[0m<br/>[37m2023-06-15 10:01:50,395 crypto.py:46         DEBUG Hash is correct.[0m<br/>[37m2023-06-15 10:01:50,471 crypto.py:20         DEBUG Computing sha512 checksum for './cache/1e0fd3af-e158-45a4-ae1a-63f62355843b' file[0m<br/>[37m2023-06-15 10:01:50,525 crypto.py:20         DEBUG Computing sha256 checksum for './cache/1e0fd3af-e158-45a4-ae1a-63f62355843b' file[0m<br/>[37m2023-06-15 10:01:50,550 crypto.py:20         DEBUG Computing sha1 checksum for './cache/1e0fd3af-e158-45a4-ae1a-63f62355843b' file[0m<br/>[37m2023-06-15 10:01:50,573 crypto.py:20         DEBUG Computing md5 checksum for './cache/1e0fd3af-e158-45a4-ae1a-63f62355843b' file[0m<br/>[32m2023-06-15 10:01:50,837 resource.py:243        INFO  Using cached artifact 'native-image-installable-svm-java11-linux-amd64-22.3.2.jar'.[0m<br/>[32m2023-06-15 10:01:50,837 resource.py:226        INFO  Copying resource 'native-image-installable-svm-java11-linux-aarch64-22.3.2.jar'...[0m<br/>[37m2023-06-15 10:01:50,843 resource.py:301        DEBUG Using md5 checksum to fetch artifacts from cacher[0m<br/>[37m2023-06-15 10:01:50,843 resource.py:311        DEBUG Using cache url 'http://ba-cekit-cacher.usersys.redhat.com/resource/9d87d8650aa17e2ec375a587dc445651'[0m<br/>[37m2023-06-15 10:01:50,843 tools.py:39         DEBUG Downloading from 'http://ba-cekit-cacher.usersys.redhat.com/resource/9d87d8650aa17e2ec375a587dc445651' as ./cache/64a57572-6826-40a9-896a-045980d7ac3e[0m<br/>[37m2023-06-15 10:01:50,856 resource.py:465        DEBUG Cannot hit artifact: 'native-image-installable-svm-java11-linux-aarch64-22.3.2.jar' via cache, trying directly.[0m<br/>[37m2023-06-15 10:01:50,856 tools.py:39         DEBUG Downloading from 'https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-22.3.2/native-image-installable-svm-java11-linux-aarch64-22.3.2.jar' as ./cache/64a57572-6826-40a9-896a-045980d7ac3e[0m<br/>Downloading native-image-installable-svm-java11-linux-aarch64-22.3.2.jar<br/>[37m2023-06-15 10:01:51,520 crypto.py:34         DEBUG Checking './cache/64a57572-6826-40a9-896a-045980d7ac3e' md5 hash...[0m<br/>[37m2023-06-15 10:01:51,520 crypto.py:20         DEBUG Computing md5 checksum for './cache/64a57572-6826-40a9-896a-045980d7ac3e' file[0m<br/>[37m2023-06-15 10:01:51,569 crypto.py:46         DEBUG Hash is correct.[0m<br/>[37m2023-06-15 10:01:51,570 crypto.py:20         DEBUG Computing sha512 checksum for './cache/64a57572-6826-40a9-896a-045980d7ac3e' file[0m<br/>[37m2023-06-15 10:01:51,617 crypto.py:20         DEBUG Computing sha256 checksum for './cache/64a57572-6826-40a9-896a-045980d7ac3e' file[0m<br/>[37m2023-06-15 10:01:51,675 crypto.py:20         DEBUG Computing sha1 checksum for './cache/64a57572-6826-40a9-896a-045980d7ac3e' file[0m<br/>[37m2023-06-15 10:01:51,725 crypto.py:20         DEBUG Computing md5 checksum for './cache/64a57572-6826-40a9-896a-045980d7ac3e' file[0m<br/>[32m2023-06-15 10:01:51,789 resource.py:243        INFO  Using cached artifact 'native-image-installable-svm-java11-linux-aarch64-22.3.2.jar'.[0m<br/>[32m2023-06-15 10:01:51,789 resource.py:226        INFO  Copying resource 'apache-maven-3.8.6-bin.tar.gz'...[0m<br/>[37m2023-06-15 10:01:51,794 resource.py:301        DEBUG Using md5 checksum to fetch artifacts from cacher[0m<br/>[37m2023-06-15 10:01:51,806 resource.py:311        DEBUG Using cache url 'http://ba-cekit-cacher.usersys.redhat.com/resource/0e07de4a7b5c84ebed737a2002f52019'[0m<br/>[37m2023-06-15 10:01:51,808 tools.py:39         DEBUG Downloading from 'http://ba-cekit-cacher.usersys.redhat.com/resource/0e07de4a7b5c84ebed737a2002f52019' as ./cache/fdf4effc-0e93-4ac2-a0e8-e42fda68edf5[0m<br/>Downloading 0e07de4a7b5c84ebed737a2002f52019<br/>[37m2023-06-15 10:01:51,836 crypto.py:34         DEBUG Checking './cache/fdf4effc-0e93-4ac2-a0e8-e42fda68edf5' md5 hash...[0m<br/>[37m2023-06-15 10:01:51,836 crypto.py:20         DEBUG Computing md5 checksum for './cache/fdf4effc-0e93-4ac2-a0e8-e42fda68edf5' file[0m<br/>[37m2023-06-15 10:01:51,852 crypto.py:46         DEBUG Hash is correct.[0m<br/>[37m2023-06-15 10:01:52,015 crypto.py:20         DEBUG Computing sha512 checksum for './cache/fdf4effc-0e93-4ac2-a0e8-e42fda68edf5' file[0m<br/>[37m2023-06-15 10:01:52,110 crypto.py:20         DEBUG Computing sha256 checksum for './cache/fdf4effc-0e93-4ac2-a0e8-e42fda68edf5' file[0m<br/>[37m2023-06-15 10:01:52,116 crypto.py:20         DEBUG Computing sha1 checksum for './cache/fdf4effc-0e93-4ac2-a0e8-e42fda68edf5' file[0m<br/>[37m2023-06-15 10:01:52,123 crypto.py:20         DEBUG Computing md5 checksum for './cache/fdf4effc-0e93-4ac2-a0e8-e42fda68edf5' file[0m<br/>[32m2023-06-15 10:01:52,148 resource.py:243        INFO  Using cached artifact 'apache-maven-3.8.6-bin.tar.gz'.[0m<br/>[37m2023-06-15 10:01:52,148 docker.py:32         DEBUG Artifacts handled[0m<br/>[32m2023-06-15 10:01:52,160 base.py:371        INFO  Rendering Dockerfile...[0m<br/>[37m2023-06-15 10:01:52,508 base.py:390        DEBUG Dockerfile rendered[0m<br/>[32m2023-06-15 10:01:52,509 builder.py:72         INFO  The --dry-run parameter was specified, build will not be executed, exiting[0m<br/>[32m2023-06-15 10:01:52,509 cli.py:487        INFO  Finished![0m<br/>[Pipeline] dir<br/>Running in /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-images/workspaces/kogito-s2i-builder/target/image<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ git rev-parse HEAD<br/>[Pipeline] sh<br/>+ docker buildx build --push --sbom=false --provenance=false --platform linux/amd64 -t localhost:5000/kogito-s2i-builder:f8220744e1c7ac2381ab7e18d85f534af4ff2722-linux-amd64 .<br/>ERROR: listing workers for Build: failed to list workers: Unavailable: failed to receive server preface within timeout<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] lock<br/>Trying to acquire lock on [https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/869/ build failed]<br/>Resource [https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/869/ build failed] did not exist. Created.<br/>Lock acquired on [https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/869/ build failed]<br/>[Pipeline] {<br/>[Pipeline] }<br/>Lock released on resource [https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/869/ build failed]<br/>[Pipeline] // lock<br/>[Pipeline] sh<br/>+ rm -rf kogito-s2i-builder_console.log<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/869/consoleText<br/>+ tail -n 400<br/>
</details>

**Comment by ricardozanini** [comment]
jenkins retest this

**Comment by kie-ci1** [comment]
**PR job** `#870` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/870/display/redirect

**Test results:**
- PASSED: 269
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/870/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Perform_a_second_incremental_s2i_build_for_native_scenario,_this_time,_with_native_enabled/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Perform a second incremental s2i build for native scenario, this time, with native enabled</a></summary>
Phrase 'Expanding artifacts from incremental build...' was not found in the output of S2I
</details>

**Comment by ricardozanini** [comment]
@radtriste I gave up Jenkins 😆

**Comment by spolti** [comment]
it is not related, we can merge

**Pull request merged**